experiments/guess-static-sites/README.md

```diff
@@ -9,9 +9,11 @@ Guess.js for non-Webpack sites.
 :heavy_check_mark: **Dynamic:** As your site changes, prefetch links will adjust accordingly. (No more hardcoded prefetch URLs.) 
 
 :heavy_check_mark: **Intelligent:** Predictive fetching uses the client's connection type to determine whether a resource should be prefetched.
-<br></br>
+
+:point_right: <a href="https://guess-static-sites-demo.firebaseapp.com/" target="_blank">View Demo</a>
```

Comment:
@khempenius Nice work. Would it be possible to push the source for this to https://github.com/guess-js?